### ISSUE TITLE
[UWP] ArgumentException on desktop with webcam

### ISSUE THREAD

**Issue by roelvdwater** [Redth/ZXing.Net.Mobile#413]
When scanning a QR code on a desktop with an attachted webcam, the Events section in the Diagnostics Tools in Visual Studio is being spammed with a `System.ArgumentException`. Also, a QR code on a paper is not being scanned and I'm pretty sure that's caused by the same error as the exception cause.

**Comment by redwolf2** [comment]
Bitmaps may raise this exception internal, when an image is loaded. If that is the case, nothing can be done. 

Please post the whole stacktrace or console log.

**Comment by Hoithebest** [comment]
In my Output window, I see the following being spammed:
```
Exception thrown: 'System.ArgumentException' in ZXingNetMobile.dll
WinRT information: The parameter is incorrect.
```
![1](https://cloud.githubusercontent.com/assets/3884615/18221137/c56cfdc0-7179-11e6-949e-edc77638240e.PNG)
Showing the image is currently not possible, but I can ensure you that I can scan and read it correctly on my phone (Lumia 950).

**Comment by Tapperi** [comment]
Any updates on this? 

I have same issue. Trying to get Xamarin Forms uwp sample working with desktop webcam.

**Comment by F1nZeR** [comment]
@Tapperi it has been fixed in ZXing.Net already and will be merged soon (#417)

**Comment by Adrianotiger** [comment]
@F1nZeR do you know when a Xamarin update will be available? Because this fix is not available in the last Xamarin NuGet package (2.1.47).

**Comment by Redth** [comment]
Thanks for reporting this issue!  Unforunately it took me way too long to respond 😭.  Sorry, I apologize!  Recently the source code for this project was completely refactored to modernize it.  Many PR's were included in this effort, and many bugs were hopefully fixed.  Please try out the latest 3.x series of NuGet packages (currently in prerelease).  To try and make the project more maintainable in my spare time going forward, I've decided to close all existing issues to start with a clean slate.  If you're still experiencing this issue on the newest version, please open a new issue with as much detail as possible.  Thank you for your patience and understanding!  Happy scanning!
